vimeo/VimeoNetworking
# VIM-6641: Adds support for Albums and VIMVideos labeled as Stock

### Ticket
[VIM-6641](https://vimean.atlassian.net/browse/VIm-6641)

### Ticket Summary
Introduce support for requesting and parsing `Album` objects, including password-protected albums. 

Add support for identifying a `VIMVideo` object as a stock video. More information about Vimeo Stock can be found at [vimeo.com/stock](https://vimeo.com/stock).

### Implementation Summary
This pull request merges previously reviewed work into VimeoNetworking's `develop` branch.
- [VIM-6642: Adds support for Albums](https://github.com/vimeo/VimeoNetworking/pull/278)
- [VIM-6642: Creates generalized method for creating password-protected requests](https://github.com/vimeo/VimeoNetworking/pull/303)
- [VIM-6849: Adds support for identifying stock videos](https://github.com/vimeo/VimeoNetworking/pull/304)


### How to Test
- Ensure the sample project builds and runs.
- Ensure unit tests continue to pass.
